server/routes/form.js

```diff
@@ -497,6 +499,21 @@ module.exports = function Index({
         })
         await requiresOpenConditions(bookingId, transactionalDbClient)
 
+        // Reset cat so it appears the categoriser originally chose open conditions!
+        const categorisationRecord = await formService.getCategorisationRecord(bookingId, transactionalDbClient)
+        const { formObject } = categorisationRecord
+        const newData = R.assocPath(
+          ['categoriser', 'provisionalCategory'],
+          {
+            suggestedCategory: userInput.supervisorOverriddenCategory,
+            categoryAppropriate: 'Yes',
+            otherInformationText: formObject.categoriser.provisionalCategory.otherInformationText,
+            sentBackOverriddenCategoryText: formObject.categoriser.provisionalCategory.overriddenCategoryText,
```

Comment:
Wouldn't mind having a quick chat about 'previousOverrideCategoryText' versus 'sentBackOverridenCategoryText' and where they are displayed.